MailChimp.Net/Models/MarketingPermissionText.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+
+namespace MailChimp.Net.Models
+{
+    public enum MarketingPermissionText
+    {
+        Email,
+        DirectMail,
+        CustomizedOnlineAdvertising
+    }
+
+
+    public static class MarketingPermissionTextHelpers
+    {
+        public static Dictionary<string, MarketingPermissionText> GetMarketingPermissions() =>
```

Comment:
Change this to a loop on the enum properties and string split on case for string - this will allow additional marketing permissions to be added in the future with no code change.